blueman/gui/manager/ManagerDeviceList.py

```diff
@@ -271,6 +271,10 @@ def on_finished(fader):
             self.row_update_event(tree_iter, "Paired", device['Paired'])
         except Exception as e:
             logging.exception(e)
+        try:
+            self.row_update_event(tree_iter, "Connected", device["Connected"])
+        except Exception as e:
```

Comment:
I just kept it the same as the other two, Trusted and Paired. But to be honest, no idea why they are there. These were blanket exceptions before and in the logging commit I at least have the logging module print the exception so we can figure out why they are there.

I can drop them if you like.